nservicebus/sagas/reply-replytooriginator-differences.md

```diff
@@ -16,7 +16,7 @@ The following diagram details a scenario where two sagas and an integration endp
 
 ![Sample sequence diagram](reply-replytooriginator-differences.png)
 
-The reason a call to `Reply<ShipOrder>()` sends a message to the `Shipment Gateway` is that it is invoked in the context of handling the `ShipmentReserved` message, and the return address of `ShipmentReserved` is `Shipment Gateway`.
+The reason a call to `Reply(new ShipOrder())` sends a message to the `Shipment Gateway` is that it is invoked in the context of handling the `ShipmentReserved` message, and the return address of `ShipmentReserved` is `Shipment Gateway`.
```

Comment:
There were a couple places where you made changes from something like `Reply<ShipOrder>()` to `Reply(new ShipOrder())`. Why make this change?